src/SegmentedArc.js

```diff
@@ -27,6 +27,7 @@ export const SegmentedArc = ({
   rangesTextStyle = styles.rangeTextStyle,
   capInnerColor = '#28E037',
   capOuterColor = '#FFFFFF',
+  alignRangesWithSegments = true,
```

Comment:
`true` as the default is a good idea. It was a bit confusing before why we would want to default to `false`.

Reply:
I had a 50/50 shot of getting it right 😉